test/unit/helpers/MappingHelpers.test.ts

```diff
@@ -0,0 +1,129 @@
+import { mapServiceDtoToService } from '../../../src/helpers/MappingHelpers';
+import { ServiceDto } from '../../../src/components/types';
+
+describe('mapServiceDtoToService', () => {
+    it('should correctly map empty serviceDto to service', () => {
+        const serviceDto: ServiceDto = {
+            name: 'ui',
+            version: '1.0.0',
+            capabilities: {},
+            expectations: {},
+        };
+
+        expect(mapServiceDtoToService(serviceDto).name).toBe(serviceDto.name);
+        expect(mapServiceDtoToService(serviceDto).version).toBe(
+            serviceDto.version
+        );
+        expect(mapServiceDtoToService(serviceDto).capabilities).toEqual({});
+        expect(mapServiceDtoToService(serviceDto).expectations).toEqual({});
```

Comment:
instead of multiples expect you could use just oneL

```
expect(mapServiceDtoToService(serviceDto)).toMatchObject(...)
```

(https://jestjs.io/docs/en/expect#tomatchobjectobject)